homeassistant/components/zha/core/const.py

```diff
@@ -118,13 +120,24 @@
 
 CONF_BAUDRATE = "baudrate"
 CONF_DATABASE = "database_path"
+CONF_DEFAULT_LIGHT_TRANSITION = "default_light_transition"
 CONF_DEVICE_CONFIG = "device_config"
+CONF_ENABLE_IDENTIFY_ON_JOIN = "enable_identify_on_join"
 CONF_ENABLE_QUIRKS = "enable_quirks"
 CONF_FLOWCONTROL = "flow_control"
 CONF_RADIO_TYPE = "radio_type"
 CONF_USB_PATH = "usb_path"
 CONF_ZIGPY = "zigpy_config"
 
+CONF_ZHA_OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_DEFAULT_LIGHT_TRANSITION): cv.positive_int,
```

Comment:
Maybe this should be a ZHA-specific thing...

Reply:
what’s the protocol here? I was under the impression that things usually start out in integrations and then back their way into core...

Reply:
There is no protocol, it has been discussed more often. If the PR title reflected the actual features in this PR, I would have responded to it way earlier.

IMHO, use the effort to solve this globally.